### PR TITLE
[Feature] Logs UI update

### PR DESCRIPTION
**Pull request by TackAdam**
### Description
1. Update the UI to conform with header changes.

Logs Before:
Screenshot 2024-08-23 at 9 39 17 AM

Screenshot 2024-08-23 at 9 38 53 AM

After:
Screenshot 2024-08-23 at 9 36 30 AM

Screenshot 2024-08-23 at 9 36 57 AM

Remade Type filter working:
Screenshot 2024-08-23 at 9 40 10 AM

PPL button adjustment:
Screenshot 2024-08-23 at 10 50 02 AM

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/event_analytics/home/home.tsx [hunk 1]
```diff
@@ -72,7 +69,7 @@ const EventAnalyticsHome = (props: IHomeProps) => {
   const history = useHistory();
   const [savedHistories, setSavedHistories] = useState<any[]>([]);
   const [selectedHistories, setSelectedHistories] = useState<any[]>([]);
-  const [isActionsPopoverOpen, setIsActionsPopoverOpen] = useState(false);
+  const [_isActionsPopoverOpen, setIsActionsPopoverOpen] = useState(false);
```
Comment (paulstn) [inline public/components/event_analytics/home/home.tsx:72]: If we're not using `_isActionsPopoverOpen`, why are we keeping it? The setter function wouldn't be needed if the getter isn't being accessed anywhere, right?

**Review by paulstn — COMMENTED** [review]
It seems like there are some test failures, one of which I looked at failed due to a required parameter not being provided

File: public/components/event_analytics/home/home.tsx (hunk 1 above)
Comment (ps48) [reply public/components/event_analytics/home/home.tsx:72]: ➕

**Review by ps48 — APPROVED** [review]

File: public/components/event_analytics/home/home.tsx (hunk 1 above)
Comment (TackAdam) [reply public/components/event_analytics/home/home.tsx]: Good catch, was used in the event explorer button before redirecting, removed it from both locations.

**Comment by TackAdam** [comment]
> It seems like there are some test failures, one of which I looked at failed due to a required parameter not being provided

CI's all passing. Checked manually for cypress changes for creating visualizations in explorer. All working correctly

**Review by paulstn — APPROVED** [review]
Looks good, thanks Adam!

**Pull request merged**